test/integration_tests/source_separation_pipeline_test.py

```diff
@@ -1,16 +1,24 @@
 import os
 import sys
 
+import pytest
 import torch
 import torchaudio
-from torchaudio.prototype.pipelines import CONVTASNET_BASE_LIBRI2MIX
+from torchaudio.prototype.pipelines import CONVTASNET_BASE_LIBRI2MIX, HDEMUCS_HIGH_MUSDB_PLUS
 
 
 sys.path.append(os.path.join(os.path.dirname(__file__), "..", "..", "examples"))
 from source_separation.utils.metrics import PIT, sdr
 
 
-def test_source_separation_models(mixture_source, clean_sources):
+@pytest.mark.parametrize(
+    "bundle,task,channel,expected_score",
```

Comment:
nit: channels

Reply:
nvm, this is following convention from https://github.com/pytorch/audio/blob/main/CONTRIBUTING.md